package.json

```diff
@@ -65,7 +65,7 @@
         "@typescript-eslint/eslint-plugin": "^6.2.0",
         "@typescript-eslint/parser": "^6.2.0",
         "commitlint": "^17.6.7",
-        "crawlee": "^3.4.2",
+        "crawlee": "^3.5.3",
```

Comment:
FYI this was wrong, the place that needs the new range is the apify package (`packages/apify/package.json`), not the root deps (as that is basically just dev deps)